ios/Cartfile

```diff
@@ -1,2 +1,2 @@
 github "stephencelis/SQLite.swift" "swift-2.3"
-github "mozilla-magnet/magnet-scanner-ios" ~> 0.3.4
+github "arcturus/magnet-scanner-ios" "betterscanning"
```

Comment:
This will change to a proper tag once https://github.com/mozilla-magnet/magnet-scanner-ios/pull/11 lands.